### DISCUSSION TITLE
Errors when opening a DEV environment on Codespaces

### DISCUSSION THREAD

**Discussion by lmilbaum** [renovatebot/renovate#22280] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

Opening an DEV environment with no errors

### What you tried so far.

Here are the errors:
```
Jest (renovate)

/bin/sh: 1: cross-env: not found
error Command failed with exit code 127.


[error] failed to retrieve test file list. TestExplorer might show incomplete test items

[info] jest process failed to start, most likely due to env or project configuration issues, please see: https://github.com/jest-community/vscode-jest/blob/master/README.md#jest-failed-to-run
```

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment]
Needs a yarn install first?

**Reply by HonkingGoose** [reply]
I think Codespaces runs a `yarn install` step when it starts the dev container. I guess Jest starts _before_ the `yarn install` is done, which leads to errors.

Maybe there's a way to tell Codespaces to only start Jest _after_ all the necessary steps are done.

**Reply by viceice** [reply]
it's probably the vscode jest addon.
